DigitalLearningSolutions.Data/Services/CandidateAssessmentDownloadFileService.cs

```diff
@@ -70,7 +70,7 @@ private static void AddSummarySheet(IXLWorkbook workbook, CandidateAssessmentExp
             rowNum++;
             sheet.Cell(rowNum, 1).Value ="Learner PRN";
             sheet.Cell(rowNum, 1).Style.Fill.BackgroundColor = XLColor.LightBlue;
-            sheet.Cell(rowNum, 2).Value = candidateAssessmentExportSummary.CandidatePrn;
+            sheet.Cell(rowNum, 2).Value = (!string.IsNullOrEmpty(candidateAssessmentExportSummary.CandidatePrn) ? "Recorded" : "Not Recorded");
```

Comment:
We want to show the candidates PRN if present, so this line should be something like:
`sheet.Cell(rowNum, 2).Value = (!string.IsNullOrEmpty(candidateAssessmentExportSummary.CandidatePrn) ? candidateAssessmentExportSummary.CandidatePrn.ToString() : "Not Recorded");`